src/beat/mod.rs

```diff
@@ -4,11 +4,12 @@ use serenity::client::Context;
 use serenity::framework::standard::CommandResult;
 use serenity::model::channel::Message;
 
-pub fn beat_command_group(
+pub async fn beat_command_group(
     ctx: &Context,
     msg: &Message,
     config: &Configuration,
     db: &DbPool,
 ) -> CommandResult {
+    msg.reply(ctx, "DEBUG: Beat command recieved").await?;
```

Comment:
use `debug!()` instead of `msg.reply()`, it logs its output instead of replying which isn't really how you want to, well, log events. You can also annotate `beat_command_group` with `#[tracing::instrument(skip_all)]` to give it a trace context for the debug message. The `skip_all` is because the input structs are big and litter the logs

Reply:
Changed!